LICENSING.md

```
@@ -1,81 +1,122 @@
-apps/chat/LICENSE                 # Apache-2.0
+## For Contributors
 
-core/lightfast/LICENSE            # Apache-2.0
-core/deus/LICENSE                 # Apache-2.0
+By contributing to Lightfast, you agree to license your contributions under the same license as the component you are contributing to (FSL-1.1-ALv2 or MIT).
+
+We may ask you to sign a Contributor License Agreement (CLA) for significant contributions.
+
+## File Structure
+
+```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)